apps/not-arrived/behaviours/radio-toggle.js

```diff
@@ -0,0 +1,16 @@
+module.exports = superclass => class extends superclass {
```

Comment:
is it worth adding a comment in your behaviour to explain the purpose of your behaviours

Reply:
Added comment describing function